tests/Backend/Mixed/WorkspacesTest.php

```diff
@@ -56,37 +43,115 @@ public function testCreateWorkspaceParam($backend)
         ]);
         $this->assertEquals($backend, $workspace['connection']['backend']);
 
-        $connection = $workspace['connection'];
-        if ($backend === self::BACKEND_SNOWFLAKE) {
-            $db = new Connection([
-                'host' => $connection['host'],
-                'database' => $connection['database'],
-                'warehouse' => $connection['warehouse'],
-                'user' => $connection['user'],
-                'password' => $connection['password'],
-            ]);
+        $backend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
 
-            $db->query("USE SCHEMA " . $db->quoteIdentifier($connection['schema']));
+        $backend->createTable("mytable", ["amount" => ($workspace['connection']['backend'] === self::BACKEND_SNOWFLAKE) ? "NUMBER" : "INT"]);
 
-            $schemaNames = array_map(function ($schema) {
-                return $schema['name'];
-            }, $db->fetchAll("SHOW SCHEMAS"));
+    }
+
+    /**
+     * @dataProvider  workspaceBackendsData
+     * @param $backend
+     */
+    public function testMixedBackendWorkspaceLoad($backend)
```

Comment:
This mixed backend stuff should be in separate test file to be able to add it to mixed backends test suite https://github.com/keboola/storage-api-php-client/blob/master/phpunit.xml.dist#L49